platooning_strategic_IHP/include/platoon_manager_ihp.h

```diff
@@ -216,6 +229,17 @@ namespace platoon_strategic
         */
         double getCurrentDowntrackDistance() const;
 
+        /**
+         * UCLA: \brief Return the platoon leader downtrack distance
```

Comment:
Add unit

Reply:
I don't see units added here.  This alone is not worth holding up the PR, but if you will be making other changes anyway, then it would be nice to see this cleaned up.